### ISSUE TITLE
CSS not injected into head

### ISSUE THREAD

**Issue by jhartma** [meteor-utilities/avatar#17]
For some reason the generated css is not injected into the head. I copied it manually into a css file so it works for now, but anyway, any ideas what could cause this?

**Comment by timothyarmes** [comment]
Hi. Did you solve this?

**Comment by jhartma** [comment]
No, I just render the blaze template into the react component for now, that works fine
